bleak/backends/bluezdbus/characteristic.py

```diff
@@ -39,17 +36,8 @@ def __init__(self, obj: dict, object_path: str, service_uuid: str):
         self.__path = object_path
         self.__service_uuid = service_uuid
 
-        # The `Handle` attribute is added in BlueZ Release 5.51. Empirically,
-        # it seems to hold true that the "/charYYYY" that is at the end of the
-        # DBUS path actually is the desired handle. Using regex to extract
-        # that and using as handle, since handle is mostly used for keeping
-        # track of characteristics (internally in bleak anyway).
-        self._handle = self.obj.get("Handle")
```

Comment:
So, basically it always got `None` here and instead always parsed it from the path instead? At least we know that it is a stable method then...

Reply:
correct